packages/fluentui/react-northstar-prototypes/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "../../../scripts/typescript/tsconfig.fluentui",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "dist/dts"
+  },
+  "paths": {
+    "@fluentui/keyboard-key": ["packages/keyboard-key/src/index"]
```

Comment:
I don't think that this is required, please check

Reply:
It is, I ran depcheck :) 
It's in chatPane, chatMessages... so on. For example: `if (keys.indexOf(getCode(e)) > -1)`